src/main/java/survey/Survey.java

```diff
@@ -39,35 +40,36 @@ public Survey(String name, Collection<Question> questions){
     }
 
     /*
-     * Sets the name
-     * @param name {String}
+     * Sets the id
+     * @param id {int}
      */
-    public void setName(String name) {
-        this.name = name;
+    public void setId(Integer id) {
+        this.id = id;
```

Comment:
Weren't these modified in a different PR?

Reply:
Yes. The PRs build on each other.

Reply:
They need to be merged in the correct order. Look at them by order of date.